Framework/Algorithms/inc/MantidAlgorithms/SANSSolidAngle.h

```diff
@@ -0,0 +1,76 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+
+#ifndef MANTID_ALGORITHMS_SANSSOLIDANGLE_H_
+#define MANTID_ALGORITHMS_SANSSOLIDANGLE_H_
+
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidAPI/Algorithm.h"
+#include "MantidAPI/SpectrumInfo.h"
+#include "MantidGeometry/Instrument/ComponentInfo.h"
+#include "MantidGeometry/Instrument/DetectorInfo.h"
+
+namespace Mantid {
+namespace Algorithms {
+/**
+
+    Performs a solid angle correction on a 2D SANS data set to correct
```

Comment:
This is confusing. First as you say this does not perform the correction, but only calculates it. Second, you can rephrase the description below to make it clear which approach is what.

Reply:
Looks like someone copied this from somewhere else. Fixed it.